ci_group/revolve2/ci_group/modular_robots_v2.py

```diff
@@ -0,0 +1,64 @@
+"""Standard modular robots."""
+import numpy as np
+from revolve2.modular_robot import ActiveHinge, Body, Brick
+from revolve2.modular_robot.v2._property_set import V2PropertySet
+
+
+def all() -> list[Body]:
+    """
+    Get a list of all standard module robots.
+
+    :returns: The list of robots.
+    """
+    return [
+        gecko_v2(),
+    ]
+
+
+def get(name: str) -> Body:
+    """
+    Get a robot by name.
+
+    :param name: The name of the robot to get.
+    :returns: The robot with that name.
+    :raises ValueError: When a robot with that name does not exist.
+    """
+    case = {
```

Comment:
While I think it reads nicely, creating the complete case always build every robot for every call to `get`. This is not very efficient. If you want to do this you could e.g. do `"gecko": gecko_v2,`
then
return `case[name]()`
so case the function pointer, but actually only call the function at the return statement.